linkerd/app/integration/src/tests/telemetry.rs

```diff
@@ -5,6 +5,7 @@
 
 mod log_stream;
 mod tcp_errors;
+mod env_json;
```

Comment:
perhaps:

```suggestion
mod env;
```

The JSON-ness of the module seems like a detail, we don't expose for log-streaming, etc.